scripts/provision-all

```diff
@@ -17,6 +17,11 @@ sudo qubesctl --show-output --skip-dom0 --targets sys-firewall state.sls sd-sys-
 echo "Set up dom0 config files, including RPC policies, and create VMs"
 sudo qubesctl --show-output state.highstate
 
+echo "Setup sd-log-buster-template vm first"
+sudo qubesctl --show-output --skip-dom0 --targets sd-log-buster-template state.sls sd-log-template-files
+# Make sure that the VM is in shutdown state
+qvm-shutdown sd-log
```

Comment:
Are you sure this is still required? When running `make all`, I see in standard out:
```
sd-log: Shutdown error: Domain is powered off: 'sd-log'
````